src/test/java/io/projectriff/functions/WordCounter.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
What is this file for? Do you use it for your testing?

Reply:
Yes. It makes a better test of “real” data. Should probably fold it into the integration tests at some point.